### ISSUE TITLE
A-Frame blog content ideas, suggestions and requests

### ISSUE THREAD

**Issue by delapuente** [aframevr/aframe#2957]
We want the content of our blog to be relevant for our community and so, we are opening this issue to track all the ideas, suggestions or requests for the content you want to see published.

When participating, think for content relevant to everybody and try to not self-promote. Use GitHub reactions to vote for the ideas you find interesting and help us to keep the community well informed.

[Jump to the conversation](https://github.com/aframevr/aframe/issues/2957#issuecomment-321643672)

### August, 7th
* [A-Frame 101](https://www.whitesmith.co/blog/a-frame-101/) introductory materials by @afvc

**Comment by ngokevin** [comment]
There's also the aframe-site repo which might be a more appropriate place: https://github.com/aframevr/aframe-site

**Comment by dmarcos** [comment]
The issue might be also relevant here. Can use this as a general call about blog post, doc materials, examples people think are not currently covered? We can file independent issues from this conversation so anybody can jump and help. Good entry points for participation.

**Comment by ngokevin** [comment]
It's something we've tried before. It's hard to follow completely through and write something well since an article is a huge commitment and technical writing is difficult. But yeah, would be great to hear about any holes.

https://github.com/aframevr/aframe/issues/2317
https://github.com/aframevr/aframe/issues/2281
https://github.com/aframevr/aframe/issues/2274

But I can say that simple examples on Glitch are most sorely needed, people have requested that. For each documentation page, we will link to relevant examples on Glitch that people can play with. Just add the Glitch links to the documentation metadata, and I'll get them visible on the website.

```
title: Cursor
examples:
  - name: Clicking
  - url: https://glitch.com/~aframe-clicking
---
```

My call to action would be to create simple Glitch examples for each documentation page to link to. Code speaks louder than words, so please share them! I'll be working on them soon myself.

**Comment by afvc** [comment]
@ngokevin @dmarcos I recently made a tutorial that has lots of little pieces of code, but in codepen, not glitch. I can pass them to glitch if you want to.
I've made code for most of the basic topics, except audio and video, I think

**Comment by Kumaresh-Rajalingam** [comment]
Awesome @afvc

**Comment by wmurphyrd** [comment]
@ngokevin in regards to contributing glitch examples, does it not matter whose glitch account they reside on, or would you want to remix them onto the aframe account and link that version in the docs?

**Comment by ngokevin** [comment]
@wmurphyrd It doesn't really matter the accounts because the Glitches can get remixed by anyone. We create examples, a contributor can remix them to add a small change and update the URL, and if we need to modify we can re-remix it.

**Comment by wmurphyrd** [comment]
@ngokevin The self-promotion from having A-Frame docs drive traffic to your personal Glitch account would be a nice incentive for contributing examples.

However, I remember one time I used a Glitch to demonstrate an issue I filed, then later accidentally deleted all the text from the HTML file in the Glitch and couldn't figure out how to restore it as it had so helpfully auto-saved the change. 

I wonder if linking to people's Glitches that can be changed or broken at any point in the future might lead to poor experiences for A-Frame docs users

**Comment by delapuente** [comment]
I'm going to gather the suggestions/requests of content per week. I'm happy to see the conversation moves forward. Thanks @afvc for stepping forward and suggest more content.

**Comment by afvc** [comment]
@delapuente maybe a tutorial that explains the differences between using AR in mobile and desktop? It's a simple thing, but for example, I was stuck on it for several days until @cwervo gave me a hand. I can make a codepen or glitch with it.

**Comment by afvc** [comment]
Also, I've been seeing loads of people that even with the help of the A-Frame docs have not managed to use obj/dae/gltf properly on blender. For several reasons : not understanding how to install gltf exporter on blender or for not having relative paths in textures and much more.
If there was a tutorial that showed two or three different ways to deal with these issues maybe less people would face them?
(I've had this problem myself for a while, with no solution yet)

**Comment by ngokevin** [comment]
Cool. Going to close as you've gotten a couple of suggestions as desired. Will leave the aframe GitHub issues for core issues or actionable items by core team or community versus a call out. Note the A-Frame blog itself has a high standard for curation of content. Helpful content should be added as now Glitches or additions to the documentation.

**Comment by ngokevin** [comment]
@afvc those look great! it would be great if you could get them into glitches and add them or send them to the docs!

**Comment by delapuente** [comment]
@ngokevin, @dmarcos and I considered this was a good idea. I don't understand why to close this issue. It was not for me only but a tool for the community to participate demanding more content.

**Comment by ngokevin** [comment]
I suggest more frictionless ways to gauge community needs. One is to be present within the community; Slack and StackOverflow are a wealth of indicators on what knowledge is needed. You can even ask for ideas there. I've also indicated the need for more forkable examples. Another good source of blog content are case studies on innovations and real projects (i.e., new components, VR UI) that you personally work on (versus writing about a topic in a vacuum). Those are good places to start versus a soapbox-y issue GitHub that might not attract attention. Thanks for being interested in it.
